salesagility/SuiteCRM
# Fixed #7723 - Fix missing campaign analysis graphs

## Description

Bring back the campaign statistics that were removed in #7663
This reverts the commits that removed it.

There exists a JSChart class, but it is missing an implementation and just returns null and empty strings everywhere, which isn't very helpful.
This brings back the Jit one.

Fixes #7723

## Motivation and Context

I'd like them back :(

## How To Test This

* View a campaign
* Click "View Status" in the top right
* The campaign stats graph should be visible

## Types of changes

### Final checklist
